R/pkg/DESCRIPTION

```diff
@@ -57,6 +57,6 @@ Collate:
     'types.R'
     'utils.R'
     'window.R'
-RoxygenNote: 5.0.1
+RoxygenNote: 6.0.1
```

Comment:
pls revert this

Reply:
Argh, I though I had removed this one. (It's really annoying when the build changes your working file set.)